numba/npyufunc/wrappers.py

```diff
@@ -280,14 +291,15 @@ def store_aligned(self, value, ind):
 
 
 class _GufuncWrapper(object):
-    def __init__(self, py_func, cres, sin, sout, cache):
+    def __init__(self, py_func, cres, sin, sout, cache, is_parfors):
```

Comment:
It'd be nice to have a comment on the significance of `is_parfors`.

Reply:
added docstr in https://github.com/numba/numba/pull/4155/commits/37684eff506bdf7fbdaa00e373d06d9321d57409